packages/ui5-proxy-middleware/src/translations/ui5-proxy-middleware.i18n.json

```
@@ -1,8 +1,8 @@
 {
     "info": {
-        "ui5VersionSource": "Using UI5 version {{version}} based on {{-source}}"
+        "ui5VersionSource": "Using SAPUI5 version: {{version}} based on: {{-source}}."
     },
     "error": {
-        "noCodeError": "Error {{-error}} thrown for request {{-request}}"
+        "noCodeError": "An error: {{-error}} was returned for the request: {{-request}}."
```

Technically `return` is the key word for a function that successfully did something and now returns a value. `Throw` is the key word for a function that had an issue and therefore throws an error (instead of returning a value) Not sure if we want to mix this up.